src/AutoMapper/Configuration/MappingExpression.cs

```diff
@@ -541,7 +541,14 @@ public void Configure(TypeMap typeMap)
                 if (attrs.Any(x => x is IgnoreMapAttribute))
                 {
                     IgnoreDestinationMember(destProperty);
-                    _reverseMap?.ForMember(destProperty.Name, opt => opt.Ignore());
+                    try
+                    {
+                        _reverseMap?.ForMember(destProperty.Name, opt => opt.Ignore());

```

Comment:
yes, it is better. I just wrote a quick fix. try-catch smells bad. I will change that as you said